hooks/pre-push

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# This script runs Android lint check before pushing your branch to remote.
+# Copy and paste it into your `projectDirectory/.git/hooks`
+# Make sure the file is executable by running `chmod +x pre-push
+
+# Check if we actually have commits to push
+commits=`git log @{u}..`
+
+if [ -z “$commits” ]; then
+ exit 0
+fi
+
+SIZE=1
+while read -r line
+do
+    echo "$line"
+    size=$(($SIZE + ${#line}))
+done < <(./gradlew lintDebug)
```

Comment:
Once the plugin is integrated, it should just be the gradle task that it comes with-- ./gradlew checkstyle

Or are you talking about something else?